example/main.tf

```diff
@@ -25,17 +25,15 @@ provider "aws" {
 }
 
 module "example_doc_db_cluster" {
-  source = "../"
-
-  namespace   = var.namespace
-  environment = var.environment
-
+  source              = "sourcefuse/arc-document-db/aws"
+  version             = "1.1.0"
```

Comment:
Don't pin the version. Add a comment saying we recommend to pin the version we aren't simply for an example reference against our latest changes.